packages/pyright-internal/src/backgroundThreadBase.ts

```diff
@@ -10,47 +10,56 @@ import { MessagePort, parentPort, TransferListItem } from 'worker_threads';
 
 import { OperationCanceledException, setCancellationFolderName } from './common/cancellationUtils';
 import { ConfigOptions } from './common/configOptions';
-import { LogLevel } from './common/console';
+import { ConsoleInterface, LogLevel } from './common/console';
 import * as debug from './common/debug';
 import { FileSystem } from './common/fileSystem';
 import { FileSpec } from './common/pathUtils';
 import { createFromRealFileSystem } from './common/realFileSystem';
 import { PyrightFileSystem } from './pyrightFileSystem';
 
+export class BackgroundConsole implements ConsoleInterface {
+    // We always generate logs in the background. For the foreground,
+    // we'll decide decide based on user setting whether.
```

Comment:
I see this is pre-existing text, so feel free to ignore this comment.

Duplicate word. Also the sentence seems to be missing the ending? "whether" what?

```suggestion
    // we'll decide based on user setting whether.
```